modAionBase/src/org/aion/base/vm/VirtualMachineSpecs.java

```diff
@@ -0,0 +1,7 @@
+package org.aion.base.vm;
+
+public final class VirtualMachineSpecs {
+
+    public static final byte AVM_VM_CODE = 0xf;
+    public static final byte FVM_DEFAULT_TX_TYPE = 0x01;
```

Comment:
`0x0f` is used when creating a new Avm contract so maybe `AVM_CREATE_CODE` is a better name

Reply:
updated both the value and the name